dev/packages/alpha/cisco/0.0.2/docs/README.md

```diff
@@ -0,0 +1,8 @@
+# Cisco Integration
```

Comment:
I assume that this file may need some more details. You can compare it other alpha packages: mysql, redis, nginx.